docs/sources/enterprise/kms-integration/_index.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "KMS integration"
+description = ""
+keywords = ["grafana", "kms", "key management system integration"]
+weight = 1200
++++
+
+# Key management systems (KMSs)
+
+You can choose to encrypt secrets stored in the Grafana database using a key from a KMS, which is a secure central storage location that is designed to help you to create and manage cryptographic keys and control their use across many services. When you integrate with a KMS, Grafana does not directly store your encryption key. Instead, Grafana stores KMS credentials and the identifier of the key, which Grafana uses to encrypt the database.
+
+Grafana integrates with the following key management systems:
+
+* AWS KMS
+* Azure Key Vault
+* [Hashicorp Vault]({{< relref "hashicorp-vault.md" >}})
```

Comment:
I'm conflicted about listing Hashicorp Vault _next to_ AWS KMS and Azure Key Vault, since they are all technically KMS integrations, but Hashicorp Vault is very different from the AWS and Azure integrations. Eventually we will add Hashicorp Vault as a proper KMS integration for database encryption, but until then I think it should remain separate from AWS and Azure. @joanlopez what do you think?

Reply:
I agree with Mitch that it's a different kind of integration. We have not added Hashicorp Vault as encryption key provider yet.